src/typeahead/typeahead.ts

```diff
@@ -80,7 +80,8 @@ let nextWindowId = 0;
     '[attr.aria-owns]': 'isPopupOpen() ? popupId : null',
     '[attr.aria-expanded]': 'isPopupOpen()'
   },
-  providers: [NGB_TYPEAHEAD_VALUE_ACCESSOR]
+  providers: [NGB_TYPEAHEAD_VALUE_ACCESSOR],
+  exportAs: 'ngbTypeahead'
```

Comment:
@ymeine, maybe just move this to the top near the selector
@pkozlowski-opensource you had some concerns about exporting typeahead or I'm mistaken?